html/Good_practices_in_Web_Dev/good-practices-in-web-dev.md

```diff
@@ -0,0 +1,248 @@
+# Good practices in Web-Dev
+
+To make sure quality user experience, your website must be displayed correctly across all the devices. To achieve this, we need to write scalable code and give the same output for all the devices.
+
+## what will it change?
+Following this will, of course, enhance the user experience once they visit your web platform. It will improve the readability of your code with the help of clear indentations, and depending upon the performance, it will also enhance the SEO rankings, pushing your platform to the top in search engines. It'll boost up the accessibility on the screen reader and other identical technologies.
+
+
+## Tell me Some examples
+
+Here you go...
+
+1. ### **Don’t Use Divs for Everything**
```

Comment:
You don't have to add stars in headings. This will be read as:

`### 1. Don’t Use Divs for Everything`